Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.yml

```diff
@@ -71,25 +71,51 @@ configuration:
   required: false
   type: 19
 description: VMware Carbon Black Enterprise EDR (formerly known as Carbon Black ThreatHunter) is an advanced threat hunting and incident response solution delivering continuous visibility for top security operations centers (SOCs) and incident response (IR) teams. (formerly known as ThreatHunter).
-display: Carbon Black Enterprise EDR
-name: Carbon Black Enterprise EDR
+display: Carbon Black Enterprise EDR Dev
+name: Carbon Black Enterprise EDR Dev
 script:
   commands:
   - arguments:
     - description: The ID of the alert to update. Get the ID from list-alerts command.
       name: alert_id
       required: true
     - auto: PREDEFINED
-      description: Workflow state to update.
-      name: state
+      description: Workflow status to update.
+      name: status
+      predefined:
+      - IN_PROGRESS
+      - CLOSED
+      - DISMISSED
+      - OPEN
+    - description: Value judgement of whether the alert(s) represent a true or false positive.
+      name: determination
+      auto: PREDEFINED
       predefined:
-      - dismissed
-      - open
+      - TRUE_POSITIVE
+      - FALSE_POSITIVE
+      - NONE
+    - description: Relative time range for the request.
+      name: time_range
+      defaultValue: -2w
+    - name: start
+      description: The lower bound of the time range. Requires end and must be a timestamp before end.
+    - name: end
+      description: The upper bound of the time range Requires start and must be a timestamp after start.
```

Comment:
Do we verify we got one OR another? Is the API returns a normal human friendly error?

Reply:
We check this in the function